packages/hyperproxy-hub-client/src/index.js

```diff
@@ -25,31 +23,4 @@ export default class HyperproxyHubClient {
             hub.once(eventName, resolve);
         });
     }
-
-    /*
-        Handle swarm event reporting
-    */
-    handleSwarmEvent() {
```

Comment:
We should keep this. Logging in experimental application is better than no log. It's P2P anyway :p

Reply:
Ok makes sense! Could you explain your last sentence?